website/docs/troubleshooting/alb/alb_fix_5.md

```diff
@@ -81,4 +81,4 @@ k8s-ui-ui-5ddc3ba496-1208241872.us-west-2.elb.amazonaws.com
 
 :::
 
-For reference, the complete set of permissions required for the AWS Load Balancer Controller can be found in the [official documentation](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/deploy/installation/#setup-iam-manually).
+For reference, the complete set of permissions required for the AWS Load Balancer Controller can be found in the [official documentation](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/deploy/installation/#setup-iam-manually).
```

Comment:
Would have been good to follow LBC_version but we use an helm chart through the terraform-aws-eks-blueprints. Since this is for reference, it's better that we follow latest. If someone see an option not available in the lab, they could update for their needs/environment.